### PR TITLE
Remove detection of strings in B018

### LINKED ISSUE #208: B018 wrongly detects inline variable or attribute docstrings
Having an inline attribute doc string or a module variable docstring, it is wrongly marked as B018 (sample from sphinx doc):

```
module_level_variable2 = 98765
"""int: Module level variable documented inline.

The docstring may span multiple lines. The type may optionally be specified
on the first line, separated by a colon.
"""

```

### PR DESCRIPTION
**Pull request by kasium**
Stops false positive for #208 issue - We need to evaluate if we can be smarter and re-add this detection in future releases.

### CODE REVIEW

File: tests/b018_classes.py [hunk 1]
```diff
@@ -30,3 +30,4 @@ class Foo3:
     123
     a = 2
     "str"
+    1
```
Comment (cooperlees) [inline tests/b018_classes.py:33]: Did we error at all without these ints? Why did you add them? Just trying to understand.

**Review by cooperlees — APPROVED** [review]
Thanks for the workaround. I'd like to release possibly today a less noisy version for people until we workout if we can re-enable strings and avoid this edge case.

I feel we could add an example string to the unittests to ensure we're actually now ignoring it. And then use it for the upcoming true fix (if we can).

File: tests/b018_classes.py (hunk 1 above)
Comment (kasium) [reply tests/b018_classes.py:33]: As you can see in the tests line 32 is ignored and to test if something is found, I added line 33

Comment (kasium) [reply tests/b018_classes.py:33]: So this is already a working test

Comment (cooperlees) [reply tests/b018_classes.py:33]: Yeah, this tests we're ignoring strs as a whole, I was more stating while we have examples that are causing the noise lets add those into this file too. But it can be next PR as well.

**Review by cooperlees — APPROVED** [review]

File: tests/b018_classes.py (hunk 1 above)
Comment (kasium) [reply tests/b018_classes.py:33]: Ah, you mean the docstring. Valid point. I'll take a look what we can improve
